snips-nlu-ontology-parsers-ffi-macros/src/builtin_entity_parser.rs

```diff
@@ -101,7 +104,11 @@ pub fn extract_entity(
     };
     let opt_filters = opt_filters.as_ref().map(|vec| vec.as_slice());
 
-    Ok(parser.extract_entities(sentence, opt_filters))
+    if reference_timestamp == i64::MIN {
```

Comment:
I would actually highly prefer if we could use `-1` for the default value, otherwise it may be impractical to use in some languages (using the ffi).